Dockerfile

```diff
@@ -16,6 +16,6 @@ FROM swaggerapi/swagger-ui:v4.13.2
 
 COPY appconfigservice/appconfig.yaml eventservice/events.yaml profileservice/profile.yaml loggingservice/logging.yaml /usr/share/nginx/html/app/
 
-ENV URLS "[{url: 'app/appconfig.yaml', name: 'App Config Building Block'}, {url: 'app/auth.yaml', name: 'Authentication Building Block'}, {url: 'app/events.yaml', name: 'Events Building Block'}, {url: 'app/profile.yaml', name: 'Profile Building Block'}, {url: 'app/logging.yaml', name: 'Logging Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/health/doc', name: 'Health Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/talent-chooser/doc', name: 'Talent Chooser Building Block'} ]"
+ENV URLS "[{url: 'app/appconfig.yaml', name: 'App Config Building Block'}, {url: 'app/auth.yaml', name: 'Authentication Building Block'}, {url: 'app/events.yaml', name: 'Events Building Block'}, {url: 'app/profile.yaml', name: 'Profile Building Block'}, {url: 'app/logging.yaml', name: 'Logging Building Block'}, {url: 'https://api.rokwire.illinois.edu/health/doc', name: 'Health Building Block'}, {url: 'https://api.rokwire.illinois.edu/talent-chooser/doc', name: 'Talent Chooser Building Block'} ]"
```

Comment:
It looks like during the previous release, the environment variables got updated. Fixing it through the below suggestion.

```suggestion
ENV BASE_URL="/docs" \
    URLS="[{url: 'app/appconfig.yaml', name: 'App Config Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/content/doc/ui/', name: 'Content Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/core/doc/ui/', name: 'Core Building Block'}, {url: 'app/events.yaml', name: 'Events Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/gr/doc/ui/', name: 'Groups Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/lms/doc/ui/', name: 'Learning Management System (LMS) Building Block'}, {url: 'app/logging.yaml', name: 'Logging Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/notifications/api/doc/ui/', name: 'Notifications Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/polls/doc/ui/', name: 'Polls Building Block'}, {url: 'app/profile.yaml', name: 'Profile Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/rewards/doc/ui/', name: 'Rewards Building Block'}, {url: 'https://api-dev.rokwire.illinois.edu/wellness/doc/ui/', name: 'Wellness Building Block'}]"
```